src/server/json_family_test.cc

```diff
@@ -223,6 +223,52 @@ TEST_F(JsonFamilyTest, SetGetFromPhonebook) {
       R"([st{stt"number":s"212 555-1234",stt"type":s"home"st},st{stt"number":s"646 555-4567",stt"type":s"office"st}s])");
 }
 
+TEST_F(JsonFamilyTest, GetBrackets) {
```

Comment:
Good that you added a lot of tests for basic behaviour 👍🏻 

I'd maybe add an intermediate one, for example a case where brackets are chained, i.e. ["a"]['b']["c"] or ["a"].b['c'], etc (if it's even allowed to mix it [] and . 🙂)